distribution/src/root/README.md

```diff
@@ -1,93 +0,0 @@
-Hazelcast Jet Readme
```

Comment:
In the proposed distribution content, I see also the readme file in it. Wdyt about adding it?

Reply:
Probably it's a leftover from the discussion, this is the final proposal - https://hazelcast.atlassian.net/browse/HZ-34

And here we agreed on removing the readme file - https://hazelcast.atlassian.net/browse/HZ-34?focusedCommentId=53183

Reply:
Based on those comments and the release pipeline changes, Ondrej opened the issue for someone from core team to address on the OS repo - https://github.com/hazelcast/hazelcast/issues/18998

Reply:
Just `release_notes.txt` is missing right now, I think it will be added later